hirola/front/templates/front/phone_category.html

```diff
@@ -15,19 +20,20 @@
         </div>
       </nav>
     </div>
-    <div class="row storage">
```

Comment:
Have a look at this one too. the class changes from 'row storage' to another one. You should have a sync with Dan on this.